src/App.js

```diff
@@ -26,30 +26,55 @@ import withNavigation from "./components/Navigation/withNavigation";
 
 // TODO: Fix ToastProvider
 
-// imports for toast wrapper 
-// import { ToastProvider } from "react-toast-notifications";
+// imports for toast wrapper
+import { ToastProvider } from "react-toast-notifications";
 
 library.add(faSearch);
 
 const SettingsWithNav = withNavigation({
   pageTitle: "Settings"
 })(Settings);
 
+const OnboardingWithNav = withNavigation({
+  iconColor: "black",
+  topNavColor: "white",
+  displayBottom: false
+})(Onboarding);
+
+const LoginWithNav = withNavigation({
+  iconColor: "black",
+  topNavColor: "white",
+  displayBottom: false
+})(Login);
+
+const RegisterWithNav = withNavigation({
+  iconColor: "black",
+  topNavColor: "white",
+  displayBottom: false
+})(Register);
+
+const DailyLogWithNav = withNavigation({
+  pageTitle: "Daily Log"
+})(DailyLog);
+
+const FoodItemWithNav = withNavigation({
```

Comment:
We gotta drill a little deeper into FoodItem to get the Navigation working (unless it's working on your branch). I got this one working on the daily-log branch tho, so no biggy.

Reply:
I got it working on my branch as well

Reply:
I wrapped FoodItem with Container instead of a <> and set the prop height and it's working well

Reply:
dope, sounds good